form-data/form-data.d.ts

```diff
@@ -8,6 +8,7 @@
 declare module "form-data" {
     class FormData {
         append(key: string, value: any, options?: any): void;
+        entries(): IteratorResult<string,Object>;
```

Comment:
Will this make it so that this package cannot be used in ES5/3? Also, the `IteratorResult` defined in `lib.es6.d.ts` only takes on generic type argument.